gensrc/script/vectorized/vectorized_functions.py

```diff
@@ -472,14 +472,24 @@
     [100014, 'last_query_id', 'VARCHAR', [], "UtilityFunctions::last_query_id"],
     [100015, 'uuid', 'VARCHAR', [], "UtilityFunctions::uuid"],
 
-    # json function
+    # json string function
     [110000, "get_json_int", "INT", ["VARCHAR", "VARCHAR"], "JsonFunctions::get_json_int",
      "JsonFunctions::json_path_prepare", "JsonFunctions::json_path_close"],
     [110001, "get_json_double", "DOUBLE", ["VARCHAR", "VARCHAR"], "JsonFunctions::get_json_double",
      "JsonFunctions::json_path_prepare", "JsonFunctions::json_path_close"],
     [110002, "get_json_string", "VARCHAR", ["VARCHAR", "VARCHAR"], "JsonFunctions::get_json_string",
      "JsonFunctions::json_path_prepare", "JsonFunctions::json_path_close"],
 
+    # json type function
+    # TODO(mofei) change type to JSON
+    [110003, "parse_json", "VARCHAR", ["VARCHAR"], "JsonFunctions::parse_json"],
+    [110004, "json_string", "VARCHAR", ["VARCHAR"], "JsonFunctions::json_string"],
+    [110005, "json_query", "VARCHAR", ["VARCHAR", "VARCHAR"], "JsonFunctions::json_query"],
+    # [110006, "json_value", "JSON", ["JSON", "VARCHAR"], "JsonFunctions::json_query"],
+    [110007, "json_exists", "BOOLEAN", ["VARCHAR", "VARCHAR"], "JsonFunctions::json_exists"],
+    [110008, "json_object", "VARCHAR", ["VARCHAR", "..."], "JsonFunctions::json_object"],
+    [110009, "json_array", "VARCHAR", ["VARCHAR", "..."], "JsonFunctions::json_array"],
+    
```

Comment:
I suggest you can comment these in this PR to avoid misusing.

Reply:
This PR is just define these functions, to shared by FE and BE, the actual implementation will be submit later, with detailed comment.

Do you mean the function comment? Or a comment to warning others not using it ?

Reply:
Finally, I decided to remove it, until a robust implementation.